tracer/src/Datadog.Trace/Ci/Tags/CodeCoverageTags.cs

```diff
@@ -0,0 +1,22 @@
+// <copyright file="CodeCoverageTags.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.Ci.Tags;
+
+/// <summary>
+/// Code Coverage span tags
+/// </summary>
+internal static class CodeCoverageTags
+{
+    /// <summary>
+    /// Test Session Code Coverage is enabled flag
+    /// </summary>
+    public const string Enabled = "test.code_coverage.enabled";
+
+    /// <summary>
+    /// Code coverage global percentage value
+    /// </summary>
+    public const string TotalLines = "test.code_coverage.lines_pct";
```

Comment:
Minor nit: I find the fact that the name for this in this tracer doesn't include `pct`(or something else that indicates it's a percentage) a little confusing, but it's also not really in scope of this change anyway.

Reply:
done in [df4c8e8](https://github.com/DataDog/dd-trace-dotnet/pull/4458/commits/df4c8e8477f2e4578973740bed056ef7f69b598f)